cmd/log-collector/cmd/root.go

```diff
@@ -33,7 +33,7 @@ const (
 	kubeconfigUsage   = "specifies the custom path for your kubeconfig"
 	kubeconfigDefault = "~/.kube/config"
 	kubeconfigShort   = "k"
-	noCleanUsage      = "no Clean Directory"
+	noCleanUsage      = "Do not Clean the Directory ( Keep both directory and Zip )"
```

Comment:
rename this flag to `keep-source-folder` or something better, `noClean` is not informative enough. It should be binary flag.. if this flag passed.. we'will keep both directory and Zip else we'll delete source-folder and keep Zip only.

Also, update documentation accordingly for this change. Review remaining flags as well and update doc if needed.

Reply:
resolved